CHANGELOG

```diff
@@ -137,6 +137,9 @@ Linear and nonlinear solvers
 - Added partially assembled convergent diagonal preconditioner for adaptively
   refined meshes (i.e. non-conforming finite element spaces), see Example 6/6p.
 
+- Added an interface to the Intel MKL Parallel Direct Sparse Solver for
+  Clusters. An example usage of the interface is shown in Example 11p.
```

Comment:
Maybe clarify that this is a parallel direct solver?

I'm not sure how many people will be familiar with this name... In fact calling it MKL CPardiso than the full name maybe more clear.